psl/parser-database/src/attributes/default.rs

```diff
@@ -381,18 +382,38 @@ fn validate_dbgenerated_args(args: &[ast::Argument], accept: AcceptFn<'_>, ctx:
     }
 }
 
-fn validate_uuid_args(args: &[ast::Argument], accept: AcceptFn<'_>, ctx: &mut Context<'_>) {
-    let mut bail = || ctx.push_attribute_validation_error("`uuid()` takes a single Int argument.");
+fn format_valid_values<const N: usize>(valid_values: &[u8; N]) -> String {
+    match valid_values.len() {
+        0 => String::new(),
+        1 => valid_values[0].to_string(),
+        2 => format!("{} or {}", valid_values[0], valid_values[1]),
+        _ => {
+            let (last, rest) = valid_values.split_last().unwrap();
+            let rest_str = rest.iter().map(|v| v.to_string()).collect::<Vec<_>>().join(", ");
```

Comment:
fwiw you can use `itertools` (which we use in `psl` already, just not in this crate) to avoid allocating an intermediate vector, but doesn't matter much either way, it's only used in an error message anyway

Reply:
Thanks, https://github.com/prisma/prisma-engines/pull/5047/commits/f9f2ef1edf4456509ade5f7125191342f7fabacd